src/hydro/hydro_driver.cpp

```diff
@@ -86,6 +88,45 @@ TaskCollection HydroDriver::MakeTaskCollection(BlockList_t &blocks, int stage) {
   // required to be 1 or blocks.size() but could also only apply to a subset of blocks.
   auto num_task_lists_executed_independently = blocks.size();
 
+  const int num_partitions = pmesh->DefaultNumPartitions();
+
+  // calculate agn triggering accretion rate
+  if ((stage == 1) &&
+      hydro_pkg->AllParams().hasKey("agn_triggering_reduce_accretion_rate") &&
+      hydro_pkg->Param<bool>("agn_triggering_reduce_accretion_rate")) {
+
+    // need to make sure that there's only one region in order to MPI_reduce to work
+    TaskRegion &single_task_region = tc.AddRegion(1);
+    auto &tl = single_task_region[0];
+    // First globally reset triggering quantities
+    auto prev_task =
+        tl.AddTask(none, cluster::AGNTriggeringResetTriggering, hydro_pkg.get());
+
+    // Adding one task for each partition. Given that they're all in one task list
+    // they'll be executed sequentially. Given that a par_reduce to a host var is
+    // blocking it's also save to store the variable in the Params for now.
+    for (int i = 0; i < num_partitions; i++) {
+      auto &mu0 = pmesh->mesh_data.GetOrAdd("base", i);
+      auto new_agn_triggering =
+          tl.AddTask(prev_task, cluster::AGNTriggeringReduceTriggering, mu0.get(), tm.dt);
+      prev_task = new_agn_triggering;
+    }
+#ifdef MPI_PARALLEL
+    auto reduce_agn_triggering =
+        tl.AddTask(prev_task, cluster::AGNTriggeringMPIReduceTriggering, hydro_pkg.get());
+    prev_task = reduce_agn_triggering;
+#endif
+
+    // Remove accreted gas
+    for (int i = 0; i < num_partitions; i++) {
+      auto &mu0 = pmesh->mesh_data.GetOrAdd("base", i);
+      auto new_remove_accreted_gas =
+          tl.AddTask(prev_task, cluster::AGNTriggeringFinalizeTriggering, mu0.get(), tm);
+      prev_task = new_remove_accreted_gas;
+    }
+  }
+
+  TaskRegion &async_region_1 = tc.AddRegion(num_task_lists_executed_independently);
```

Comment:
Any reason for this new region?

Reply:
I'm guessing this is left over changes from merging in `main` several times.